README.md

```diff
@@ -2,19 +2,23 @@
 
 http://mergely.com
 
-Mergely is a javascript component to diff/merge files interactively in a browser, providing rich API that enables you to easily integrate Mergely into your existing web application.  It is suitable for comparing text files online, e.g. .txt, .html, .xml, .c, .cpp, .java, etc.  Mergely has a javascript implementation of the Longest Common Subsequence diff algorithm (LCS) and a customizable markup engine.
+Mergely is a javascript component to differencing and merging files interactively in a browser, providing rich API that enables you to easily integrate Mergely into your existing web application. It is suitable for comparing text files online, for example, .txt, .html, .xml, .c, .cpp, .java, etc.  
```

Comment:
There is something funky with the verb "to differencing and merging", and it also loses the technical jargon "diff", which is important to retain.

```suggestion
Mergely is a JavaScript component for differencing and merging files interactively in a browser (diff/merge), providing rich API that enables you to easily integrate Mergely into your existing web application. It is suitable for comparing text files online, for example, .txt, .html, .xml, .c, .cpp, .java, etc.  
```

Reply:
Thanks for the tech explanation :)